apps/native/app/src/messages/en.ts

```diff
@@ -490,17 +490,17 @@ export const en: TranslatedMessages = {
   'edit.phone.inputlabel': 'Phone number',
   'edit.phone.button': 'Save',
   'edit.phone.button.empty': 'Save empty',
-  'edit.phone.button.error': 'Error',
-  'edit.phone.button.errorMessage': 'Could not send verification code',
+  'edit.phone.error': 'Error',
```

Comment:
Do we want to transition into some `common` namespaces for these cases that are always the same?